adapter/config/default_config.go

```diff
@@ -146,6 +146,12 @@ var defaultConfig = &Config{
 			AllowPartialMessage: false,
 			PackAsBytes:         false,
 		},
+		AwsLambda: awsLambda{
+			Enabled:            false,
+			AwsRegion:          "",
+			PayloadPassthrough: true,
+			InvocationMode:     "SYNCHRONOUS",
```

Comment:
Can we define invocation mode in lower case and convert it to upper case when assigning to a variable ?